azkaban-web-server/src/web/js/azkaban/util/date.js

```diff
@@ -94,6 +94,22 @@ var getTwoDigitStr = function(value) {
   return value;
 }
 
+// Verify if a cron String meets the requirement of Quartz Cron Expression.
+var validateQuartzStr = function (str){
+  var res = str.split(" "), len=res.length;
+
+  // A valid Quartz Cron Expression should have 6 or 7 fields.
+  if(len<6 || len>=8) return "NUM_FIELDS_ERROR";
```

Comment:
Googled, and didn't find any benefits for defining variables as const in this case. const was brought by recent ECMAScript. Might not be accepted in all browsers.